### ISSUE TITLE
Incompatibility with scikit-learn 1.0 in latest release

### ISSUE THREAD

**Issue by lunik1** [trent-b/iterative-stratification#18]
As of `scikit-learn` 1.0 the deprecation warning fixed in 0a108bc2062fd32f98c9a6305508ea213292ba08 has become a hard error. Could a new release be pushed to pypi in order to remain compatible with the latest scikit-learn?

For other users experiencing this issue (it will look something like
```
, in __init__
    super(MultilabelStratifiedShuffleSplit, self).__init__(
TypeError: __init__() takes from 1 to 2 positional arguments but 5 were given
```
^this) the workaround is to use the latest `master` of this package.

**Comment by CryptoSalamander** [comment]
I fixed this issue and make PR.

**Comment by lunik1** [comment]
This issue is already fixed on `master`, but not in the latest release. I am asking for a new release so the fix can be in a released version.

**Comment by trent-b** [comment]
Thank you for bringing this to my attention. I will aim to create a new release and upload it to pypi.

**Comment by trent-b** [comment]
Version 0.1.7 with a fix to support `scikit-learn` 1.0 has been uploaded to pypi.
